### PR TITLE
Create arkivoc.csl

### PR DESCRIPTION
**Pull request by singhfcy**
Create arkivoc.csl

### CODE REVIEW

**Review by singhfcy — COMMENTED** [review]
updated as per requirement

**Review by singhfcy — COMMENTED** [review]
removed duplicate issn

**Comment by singhfcy** [comment]
I am not able to make this change, please help

File: arkivoc.csl [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+    <sort>
+      <key variable="citation-number"/>
+    </sort>
+    <layout delimiter="," vertical-align="sup">
+      <text variable="citation-number"/>
+    </layout>
+  </citation>
+  <bibliography second-field-align="flush" entry-spacing="0">
+    <layout suffix=".">
+      <text variable="citation-number" suffix="."/>
```
Comment (bwiernik) [inline arkivoc.csl:96]:
```suggestion
  <bibliography second-field-align="margin" entry-spacing="0">
    <layout suffix=".">
      <text variable="citation-number" suffix=". "/>
```

**Comment by adam3smith** [comment]
What did you run locally? 
And do you use a fork or checkout this repo?

**Comment by bwiernik** [comment]
I pulled master from the CSL repo, then I checked out this PR using the button on the webpage (it checks out the fork's branch). I rebased the PR branch onto the CSL repo master. Then git diff shows just one file changed. Then I pushed the rebased repo to update the PR/fork branch. That left it showing 4 files changed (even though the changes shown for the other 3 are already in master--coming from 3 recently merged PRs)

**Pull request merged**